devops/vm-fsicos3-dokku.yml

```diff
@@ -1,4 +1,10 @@
 - hosts: fsicos3
+  vars:
+    dokku_wildcard_domains:
+      - '*.dokku.fsicos3.icos-cp.eu'
+      - '*.app.icos-cp.eu'
+    dokku_static_domains:
+      - 'curve.icos-ri.eu'
```

Comment:
It would be nice to define `curve.icos-ri.eu` or `curve.app.icos-cp.eu` as the main domain, and redirect the other one to it.